core/tee/tee_svc.c

```diff
@@ -437,23 +424,40 @@ const struct tee_props tee_propset_tee[] = {
 	 */
 };
 
+__weak const struct tee_props *vendor_propset_client = NULL;
+__weak const unsigned long vendor_propset_client_len = 0;
+__weak const struct tee_props *vendor_propset_ta = NULL;
+__weak const unsigned long vendor_propset_ta_len = 0;
+__weak const struct tee_props *vendor_propset_tee = NULL;
```

Comment:
I would prefer a struct containing both the pointer and the length like:
```
struct tee_vendor_props {
    const struct tee_props *props;
    size_t len;
};
```
To guarantee that both pointer and length is overridden at the same time.
By the way, there's no need to assign `0` and `NULL` that's done automatically.